src/ObjectUtils/AddressTest.cpp

```diff
@@ -27,6 +27,24 @@ TEST(Address, SymbolVirtualAddressToAbsoluteAddress) {
                "Check failed");
 }
 
+TEST(Address, SymbolAbsoluteAddressToVirtualAddress) {
+  EXPECT_EQ(SymbolAbsoluteAddressToVirtualAddress(0x1010, 0x1000, 0, 0), 0x10);
+  EXPECT_EQ(SymbolAbsoluteAddressToVirtualAddress(0x2010, 0x2000, 0x1000, 0), 0x1010);
+  EXPECT_EQ(SymbolAbsoluteAddressToVirtualAddress(0x1100, 0x1000, 0, 0xFF), 0x100);
+  EXPECT_EQ(SymbolAbsoluteAddressToVirtualAddress(0x5100, 0x5000, 0x1000, 0x10FF), 0x2100);
+
+  // Invalid input
+  EXPECT_DEATH((void)SymbolAbsoluteAddressToVirtualAddress(0x5100, 0x5001, 0x1000, 0x10FF),
```

Comment:
An alternative to `(void)` that I recently learned about is `std::ignore = ...`. I do not really care if you change it or not, just thought maybe you dont know about it.

Reply:
Hehe thanks, I was copy-pasting. I changed to `std::ignore` throughout the file since I prefer it.